alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/DestinationTests.kt

```diff
@@ -135,7 +135,7 @@ class DestinationTests : ESTestCase() {
     }
 
     fun `test slack destination create using stream`() {
-        val chimeDest = Destination("2345", 1L, 2, DestinationType.SLACK, "TestSlackDest",
+        val chimeDest = Destination("2345", 1L, 2, 1, 1, DestinationType.SLACK, "TestSlackDest",
```

Comment:
The variable is a little inaccurate, including the one for "custom webhook" in below test, but looks like it's a change in a past PR.

Reply:
thanks for pointing that out. I fixed them

Reply:
Thank you 👍